### DISCUSSION TITLE
GitHub Support Taking Forever

### DISCUSSION THREAD

**Discussion by GameVerseMaster** [community/community#123809] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

General

Its been another month since I opened another ticket, 5 months since I sent in my original ticket by email. My main account was hacked, and the phone number I own I believe is still connected to it. However the email and password associated with the account has been changed, I desperately need access to my account. Please github check my ticket! (#2705814) Im wondering if I could use the phone number as verification, to regain access to my account.

I am starting to grow impatient, this is urgent, I need access to my account. GitHub. Respond.

**Comment by ebndev** [comment]
Hi @GameVerseMaster,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
